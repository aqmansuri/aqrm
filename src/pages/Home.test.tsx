import { render } from '@testing-library/react';
import Home from './Home';

describe('<Home />', () => {
  it('displays the header', () => {
    const { container } = render(<Home />);
    expect(container).toHaveTextContent(/Blank/);
  });

  it('renders consistently', () => {
    const { asFragment } = render(<Home />);
    expect(asFragment()).toMatchSnapshot();
  });
});
