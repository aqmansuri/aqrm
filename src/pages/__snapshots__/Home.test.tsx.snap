// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> renders consistently 1`] = `
<DocumentFragment>
  <div
    class="ion-page"
  >
    <ion-header>
      <ion-toolbar>
        <ion-title>
          Blank
        </ion-title>
      </ion-toolbar>
    </ion-header>
    <ion-content
      fullscreen="true"
    >
      <ion-header
        collapse="condense"
      >
        <ion-toolbar>
          <ion-title
            size="large"
          >
            Blank
          </ion-title>
        </ion-toolbar>
      </ion-header>
      <div
        class="container"
      >
        <strong>
          Ready to create an app?
        </strong>
        <p>
          Start with Ionic 
          <a
            href="https://ionicframework.com/docs/components"
            rel="noopener noreferrer"
            target="_blank"
          >
            UI Components
          </a>
        </p>
      </div>
    </ion-content>
  </div>
</DocumentFragment>
`;
